### ISSUE TITLE
Help|About box

### ISSUE THREAD

**Issue by cdburke** [jsoftware/j-playground#84]
I added a Help|About box which includes a version number. We can tell users that they need to be at a given version number for testing, and if necessary first clear the browser cache. Feel free to improve the styling.

Here, the version number is hard coded at the top of the start.js script. We can set this manually, but it would much nicer if it were somehow auto incremented on a github push, and not part of the source scripts. Is this possible?

**Comment by joebo** [comment]
This looks like a possible solution https://github.com/marketplace/actions/bump-versions ... It would be part of the github action to update a VERSION file, which could be queried from about box using fetch()

I found some other references that looked like they could be modified to change start.js on commit (https://towardsdatascience.com/setup-version-increment-and-automated-release-process-591d87ea1221) but I didn't read all the way through

Maybe the easiest would be to have some command line utility that updated start.js which could be run during the deploy and eventually part of the babel process (or equivalent) to consolidate the and minify the JS files

**Comment by cdburke** [comment]
Thanks, just updating a Version file would work.

I originally didn't like the idea because it would mean that as soon as you push to the server, then your local copy is out of date. But since it is only the Version file, this hardly matters.

**Comment by cdburke** [comment]
In testing, updating the version file in github actions works fine.

The main drawback is that having done a push, then if you make more changes and do a another push you get a error message that "Updates were rejected because the remote contains work that you do not have locally..." . So you need to do a pull first.

Is this OK?

**Comment by joebo** [comment]
Yes, I think it's OK. I find it good practice to always pull
before starting to make a change anyways to pick up the latest changes in
the repo

**Comment by cdburke** [comment]
Done. The playground version is now 1.006.
